elide-datastore/elide-datastore-aggregation/pom.xml

```diff
@@ -0,0 +1,123 @@
+<!--
```

Comment:
You will need to update the elide-datastore/pom.xml to add a module for this new build component.